doc/UserGuide.md

```diff
@@ -827,6 +834,29 @@ When implementing a shrinking method, one has to be careful to only return
 loop. ScalaCheck has implicit shrinking methods for common types such as integers
 and lists.
 
+If the generator for a type is restricting the range of valid values by
+construction or using `Gen.suchThat`, the values that fail tests can still be
+shrunk without checking the condition. To avoid that, use `Shrink.suchThat`
+with the condition to be maintained:
+
+```scala
+/** Generate lists of an even length */
+val genEvenList: Gen[List[Int]] = Gen.sized { size =>
+  Gen.listOfN(size * 2, arbitrary[Int])
+}
+
+/** Shrink a list, maintaining even length */
+val shrinkEvenList: Shrink[List[Int]] =
+  implicitly[Shrink[List[Int]]].suchThat(_.length % 2 == 0)
+```
+
+Note that if a property fails on a value generated through `suchThat`, and is
+later shrunk (see [test case minimisation](#test-case-minimisation) below),
+the value ultimately reported as failing might not satisfy the condition given
+to `suchThat`, although it doesn't change the fact that there _exists_ a
+failing case that does. To avoid confusion, the corresponding shrink for the
+type can use `suchThat` method too.
+
```

Comment:
I think there's an extra copy paste here?  Feel free to edit your intermediate commits witih rebase.  Something strange is happening when you're trying to fix the paren for the "see below".

Reply:
I originally duplicated it because it made sense both here and above, but now I've incorporated some changes into the paragraph above the example instead.